supporting_content/dimensions/skupriceid.md

```diff
@@ -0,0 +1,39 @@
+## Example provider mappings 
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                | Column                             |
+| ------------ | --------------------------- | -------------------------------------- |
+| AWS          | CUR                         | pricing/rate\_code \| pricing/rate\_id |
+| Azure        | Cost details export or API  | ProductID                              |
+| GCP          | BigQuery Billing Export            | Not available, but can be derived from sku.id and price.tier_start_amount                                 |
+| OCI          | Cost Reports                | Not available (no price level ID)    |
+
+
+## Example scenarios for current provider data
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Scenario               | Pattern                                                              |
+| ------------ | -------------------------- | ------------------------------------------------------------------------ |
+| AWS          | CUR                        | rate\_code: KF338J7FCKZPUBD9.JRTCKXETXF.6YS6EN2CT7 rate\_id: 20457007287 |
+| Azure        | Cost Details export or API | DZH318Z0BNZH006F                                                                         |
```

Comment:
It's probably worth calling out that neither Azure nor GCP has a unique ID available to the public today.

Reply:
Okay, I revised that chunk; let me know if it's alright now.

Reply:
I think it's laid out now the way it needs to be... ping me if not.